vipps-checkout-api-faq.md

```diff
@@ -1,7 +1,21 @@
-#FAQ
+# FAQ
 
-**How are subsequent transaction operations (Capture/Cancel/refund) handled**
+### How are subsequent transaction operations (Capture/Cancel/refund) handled
 
 All subsequent transaction operations are fully supported in the Vipps epayment API. As described [here](https://github.com/vippsas/vipps-epayment-api)
 
-Please feel free to make a PR with a question if something is unclear.
\ No newline at end of file
+Please feel free to make a PR with a question if something is unclear.
+
+## When paying by card
+
+### What card types are supported?
+
+Visa and Mastercard. This includes any cards that are co-branded with VISA and Mastercard.
```

Comment:
should we specify here that cards must be enabled for online purchase as well? could even merge in the visa electron part